build.yaml

```diff
@@ -2256,6 +2256,7 @@ libs:
   language: csharp
   src:
   - src/csharp/ext/grpc_csharp_ext.c
+  - src/csharp/ext/std++compat.cc
```

Comment:
why is this only required for C#?

Reply:
All languages having a binary distribution has this problem but they have their own solution. Python, for example, uses manylinux to build python binaries and Ruby uses a static linking.